File: src/AspNetToDoApi/AspNetToDoApi/Controllers/ToDoController.cs
using AspNetToDoApi.DAL;
using AspNetToDoApi.Models;
using System;
using System.Collections.Generic;
using System.Web.Http;

namespace AspNetToDoApi.Controllers
{
    public class ToDoController : ApiController
    {
        private static ToDoRepository _toDoRepository = new ToDoRepository();
        

    }
}

File: src/JavaToDoApi/src/main/java/com/fils/controllers/ToDoController.java
package com.fils.controllers;

import com.fils.dal.ToDoRepository;
import com.fils.models.ToDoItem;
import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.*;

import java.util.List;
import java.util.UUID;

/**
 * Created by radu on 22/11/2016.
 */

@RestController
public class ToDoController {

    private ToDoRepository _todoRepository = new ToDoRepository();



}
